### PR TITLE
feat: Added Eslint + prettier

### PR DESCRIPTION
**Pull request by zain-jaffri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gg-fe",
   "version": "0.1.0",
-  "private": true,
+  "private": false,
```
Comment (shams-ali) [inline package.json:4]: you should not have yarn.lockand package-lock - either you have yarn.lovck or either you have package-lock - looks like you are using both npm and yarn - dont use both - either use on or the other - delete this file - delete node_modules - reinstall using yarn - ensure everything works - then commit and push

Comment (zain-jaffri) [reply package.json:4]: Done

File: package.json [hunk 2]
```diff
@@ -34,5 +34,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "devDependencies": {
+    "prettier": "^3.0.3"
```
Comment (shams-ali) [inline package.json:39]: you installed this but youre note using it

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
-import React from 'react';
```
Comment (shams-ali) [inline src/index.js:1]: where is the typescript?

**Pull request merged**